srclibs.cmake

```diff
@@ -192,7 +192,7 @@ if (WIN32)
         ${LIB_DIR}/breakpad/src/client/windows/handler/exception_handler.cc
         ${LIB_DIR}/breakpad/src/client/windows/handler/exception_handler.h
     )
-elseif (LINUX)
+elseif (LINUX OR FREEBSD)
```

Comment:
Does Breakpad really work? I'd be shocked.

Reply:
Right, it does not, removed.